docs/resources/new_in_doc.md

```diff
@@ -6,6 +6,28 @@ description: Overview of major recent additions to Ibexa DXP documentation.
 
 This page contains recent highlights and notable changes in Ibexa DXP documentation.
 
+## June 2023
+
+### Personalization
+
+- [Email triggers in User Guide](https://doc.ibexa.co/projects/userguide/en/master/personalization/triggers/)
```

Comment:
```suggestion
- [Email triggers](https://doc.ibexa.co/en/master/personalization/integrate_recommendation_service/#send-emails-with-recommendations)
- [Email triggers in user documentation](https://doc.ibexa.co/projects/userguide/en/master/personalization/triggers/)
```
We have two ways of referring to the user guide, with "user documentation" prevailing. I suggest we decide on one and use it consistently.